src/auth-service/routes/api-v1.js

```diff
@@ -290,7 +290,25 @@ router.put(
           return ObjectId(value);
         }),
       body("chartTitle").if(body("chartTitle").exists()).notEmpty().trim(),
-      body("period").if(body("period").exists()).notEmpty().trim(),
+      body("period")
+        .if(body("period").exists())
+        .notEmpty()
```

Comment:
For the update of defaults endpoint (PUT), this particular validation for the` period `field is misbehaving. But shall continue investigating it tomorrow. @ja-odur

Reply:
this was also resolved